packages/dev/gui/src/2D/controls/control.ts

```diff
@@ -2323,6 +2323,27 @@ export class Control implements IAnimatable {
         this.getDescendants().forEach((child) => child._markAllAsDirty());
     }
 
+    public clone(host?: AdvancedDynamicTexture): Control {

```

Comment:
missing ts doc

Reply:
Thanks for the reminder I ended up forgetting!